.github/workflows/extract-translation-source-files.yml

```diff
@@ -125,8 +152,13 @@ jobs:
           fi
           # remove translations/${{ matrix.repo }}/.git so we don't commit a submodule
           rm -rf .git
+
+      # All repositories except for edx-platform follows the standard Django i18n pattern
```

Comment:
```suggestion
```

This will be no longer true when https://github.com/openedx/edx-platform/pull/33368 is merged.